incubator/honeydipper/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+appVersion: "latest"
```

Comment:
The `appVersion` should be removed or reflect the apps version (e.g., 0.1.8)

Reply:
Thanks.